plot-api/src/commonMain/kotlin/jetbrains/letsPlot/Env.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
```

Comment:
Move to package jetbrains.letsPlot.intern.settings.

Reply:
Done